webcompat/api/endpoints.py

```diff
@@ -124,17 +124,11 @@ def get_issue_category(issue_category):
     elif issue_category == 'closed':
         params['state'] = 'closed'
         return api_request('get', issues_path, params=params)
-    # Note that 'new' here is primarily used on the homepage.
-    # For paginated results on the /issues page, see /issues/search/new.
+    # Note that 'needstriage' here is primarily used on the homepage.
+    # For paginated results on the /issues page,
+    # see /issues/search/needstriage.
```

Comment:
Can we change this comment to explain why we send a 301 now? Something like "We abort with 301 here because the new endpoint has been replaced with needstriage"`